app/models/claim/claims_preventing_payment_finder.rb

```diff
@@ -22,14 +22,20 @@ def claims_preventing_payment
     private
 
     def find_claims_preventing_payment
-      payrollable_claims_from_same_claimant = Claim.payrollable.where(teacher_reference_number: claim.teacher_reference_number)
+      eligibility_ids = [
```

Comment:
Suggestion: I think this is likely to be overlooked and may be best configured in the policies. Also all of the new policies being added won’t have a TRN so we should create a new ticket to decide on a solution for matching payees.